CHANGELOG.md

```diff
@@ -4,8 +4,10 @@
 
 ### Changed
 
-- [RetryPlugin] Renamed the configuration options for the exception retry callback from `decider` to `exception_decider`
+- RetryPlugin: Renamed the configuration options for the exception retry callback from `decider` to `exception_decider`
   and `delay` to `exception_delay`. The old names still work but are deprecated.
+- AddPathPlugin does not add prefix if the prefix already is defined. 
```

Comment:
maybe: "AddPathPlugin: Do not add the prefix if the URL already has the same prefix."